src/Haskell/Ide/Engine/Swagger.hs

```diff
@@ -81,9 +81,9 @@ mkParamsSchema fileSchemaRef textSchemaRef allParams = s
     requiredParams = concatMap isRequired allParams
 
     isRequired :: ParamDescription -> [Data.Swagger.ParamName]
-    isRequired pd = if pRequired pd == Required
-                       then [pName pd]
-                       else []
+    isRequired pd = case pRequired pd of
```

Comment:
Sorry for being pedantic, but imho it looks prettier to just pattern match on the left of `=` instead of introducing a case statement.